test-utils/utils/BaseSpec.scala

```diff
@@ -69,4 +71,45 @@ abstract class BaseSpec
     running(_ => applicationBuilder)(block)
 
   def urlEncode(input: String): String = URLEncoder.encode(input, "utf-8")
+
+  @deprecated("behave word has been replace with act word - behave.like becomes act.like")
+  override val behave: BehaveWord = new BehaveWord
```

Comment:
Does this line need to be here?

Reply:
Yes so the only was I can discourage use of behave is to deprecate it, I originally wanted to throw an exception if it was used but behave in the super class is declared with a val so it must be a val in the subclass which doesn't work for exceptions, the only other thing I can do to let the developer know they have done something wrong is to deprecate the value